### PR TITLE
add notes to plan-restricted features

### LINKED ISSUE #206: Add notes to plan-restricted features

### PR DESCRIPTION
**Pull request by aviv**
closes #206

pretty barebones- are these notes sufficient? not sure if we want to
provide more information here, or link to the pricing page maybe.

### CODE REVIEW

**Comment by ajb** [comment]
Let's link to the pricing page. Also, for features available on the plus plan, it should say "plus plan or higher".

File: contents/articles/screendoor/plans_and_billing/limitations.md [hunk 1]
```diff
@@ -11,3 +11,9 @@ Depending on your plan, Screendoor will limit the following items:
 | Collaborators | You're allowed to have this number of active [collaborators](../projects/collaborators.html) at any given time. If someone collaborates on multiple projects within your organization, they will only count as one collaborator. Collaborators for archived projects are not counted. |
 
 Once your plan's limit for a certain feature has been reached (for example, you've maxed out on projects), you won't be able to add any more unless you upgrade your plan or remove some. Remember, you can [archive inactive projects](/articles/screendoor/projects/archiving_a_project.html#archiving-a-project) so that they don't count towards your project limit.
+
+Also, you'll need to sign up for a Plus/Enterprise plan in order to use the following features:
+
+- Electronic signatures
+- Advanced collaboration features
```
Comment (ajb) [inline contents/articles/screendoor/plans_and_billing/limitations.md]: Can we make it clear here that "Advanced collaborator features" == project teams?

File: contents/articles/screendoor/plans_and_billing/limitations.md [hunk 2]
```diff
@@ -11,3 +11,9 @@ Depending on your plan, Screendoor will limit the following items:
 | Collaborators | You're allowed to have this number of active [collaborators](../projects/collaborators.html) at any given time. If someone collaborates on multiple projects within your organization, they will only count as one collaborator. Collaborators for archived projects are not counted. |
 
 Once your plan's limit for a certain feature has been reached (for example, you've maxed out on projects), you won't be able to add any more unless you upgrade your plan or remove some. Remember, you can [archive inactive projects](/articles/screendoor/projects/archiving_a_project.html#archiving-a-project) so that they don't count towards your project limit.
+
+Also, you'll need to sign up for a Plus/Enterprise plan in order to use the following features:
+
+- Electronic signatures
```
Comment (ajb) [inline contents/articles/screendoor/plans_and_billing/limitations.md]: Let's add links to the docs for each of these

File: contents/articles/screendoor/plans_and_billing/limitations.md [hunk 3]
```diff
@@ -11,3 +11,9 @@ Depending on your plan, Screendoor will limit the following items:
 | Collaborators | You're allowed to have this number of active [collaborators](../projects/collaborators.html) at any given time. If someone collaborates on multiple projects within your organization, they will only count as one collaborator. Collaborators for archived projects are not counted. |
 
 Once your plan's limit for a certain feature has been reached (for example, you've maxed out on projects), you won't be able to add any more unless you upgrade your plan or remove some. Remember, you can [archive inactive projects](/articles/screendoor/projects/archiving_a_project.html#archiving-a-project) so that they don't count towards your project limit.
+
+Also, you'll need to sign up for a Plus/Enterprise plan in order to use the following features:
```
Comment (ajb) [inline contents/articles/screendoor/plans_and_billing/limitations.md]: This information is duplicative, and is liable to go out of date. I would change to say:

> In addition, certain advanced features are only available on certain plans:

File: contents/articles/screendoor/plans_and_billing/limitations.md (hunk 1 above)
Comment (aviv) [inline contents/articles/screendoor/plans_and_billing/limitations.md]: :white_check_mark: I just replaced the "Advanced collaborator features" as it's really just marketing copy and doesn't belong here, IMO

**Comment by aviv** [comment]
@kariannemah: care to take a quick look? Want to make sure this is as clear as possible.

File: contents/articles/screendoor/plans_and_billing/limitations.md [hunk 4]
```diff
@@ -11,3 +11,11 @@ Depending on your plan, Screendoor will limit the following items:
 | Collaborators | You're allowed to have this number of active [collaborators](../projects/collaborators.html) at any given time. If someone collaborates on multiple projects within your organization, they will only count as one collaborator. Collaborators for archived projects are not counted. |
 
 Once your plan's limit for a certain feature has been reached (for example, you've maxed out on projects), you won't be able to add any more unless you upgrade your plan or remove some. Remember, you can [archive inactive projects](/articles/screendoor/projects/archiving_a_project.html#archiving-a-project) so that they don't count towards your project limit.
+
+In addition, some advanced features are only available on certain plans:
+
+- [Electronic signatures](../response_form/signatures.html)
+- [Project teams](../projects/teams.html)
+- [Open Data Sync](../integrations/socrata.html)
```
Comment (kariannemah) [inline contents/articles/screendoor/plans_and_billing/limitations.md]: We're calling this "Socrata data sync," right?

**Comment by kariannemah** [comment]
Looks good! One question about naming, but that's it!

File: contents/articles/screendoor/plans_and_billing/limitations.md (hunk 4 above)
Comment (aviv) [inline contents/articles/screendoor/plans_and_billing/limitations.md]: Yep, ty! Better to use the KB language than the splash language here.

**Pull request merged**